locksmith/src/websub/jobs/keys.ts

```diff
@@ -97,6 +116,284 @@ async function notifyHooksOfAllUnprocessedKeys(hooks: Hook[], network: number) {
   }
 }
 
+const getRecipient = async (lockAddress: string, ownerAddress: string) => {
+  const metadata = await userMetadataOperations.getMetadata(
+    lockAddress,
+    ownerAddress,
+    true // include protected
+  )
+
+  const protectedData = Normalizer.toLowerCaseKeys({
+    ...(metadata?.userMetadata?.protected ?? {}),
+  })
+
+  return protectedData?.email as string
+}
+
+const getMembershipStatus = async ({
```

Comment:
Move to membershipOperations?

Reply:
Also `getMembershipStatus` is not very explicit. Maybe `getMembershipState` because it returns a bunch of values about the membership